### PR TITLE
61 center body content

### PR DESCRIPTION
**Pull request by loganhochwald**
Added padding around nav bar, moved card relative to nav bar downwards

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -26,12 +26,12 @@ export default function RootLayout({
 }) {
   return (
     <html id="html" lang="en">
-      <body className={ inter.className }>
```
Comment (colinwilliams91) [inline app/layout.tsx:29]: looks like this got un-linted. Maybe you have an auto-format on save?
try this:
`ctrl + shift + p` --> open settings, not the json one.
--> search "format".
--> you should see `Editor: format on save mode`
select --> "modificationsIfAvailable" in the dropdown
```suggestion
      <body className={ inter.className }>
```

File: app/layout.tsx [hunk 2]
```diff
@@ -26,12 +26,12 @@ export default function RootLayout({
...
+      <body className={inter.className}>
```
Comment (colinwilliams91) [inline app/layout.tsx:29]: Here is your incoming change

**Review by colinwilliams91 — CHANGES_REQUESTED** [review]
I like it @loganhochwald thank you! the padding on the nav and `<div>` in `app/page.tsx` are great.
The linting seemed to get messed up and im not sure why. I left a comment about auto-formatting. If it isn't linted in your code yet could you go ahead and run `npm run lint -- --fix` and then push that new comit with the linted code up?

File: app/layout.tsx (hunk 2 above)
Comment (loganhochwald) [reply app/layout.tsx]: Sent over the fixed linting.

**Review by colinwilliams91 — APPROVED** [review]
thanks Logan, looks good to me (LGTM to add to your alphabet soup) :thumbsup:

**Pull request merged**